include/swift/AST/DiagnosticsModuleDiffer.def

```diff
@@ -54,6 +54,8 @@ ERROR(decl_new_attr,none,"%0 is now %1", (StringRef, StringRef))
 
 ERROR(decl_reorder,none,"%0 in a fixed layout type changes position from %1 to %2", (StringRef, unsigned, unsigned))
 
+ERROR(decl_added,none,"%0 is added to a non-resilient type", (StringRef))
```

Comment:
Please change the wording to be consistent (either always call them non-resilient, or fixed layout/frozen)

Reply:
I've seen new diags like "cake2: Var fixedLayoutStruct.lazyD.storage is added to a non-resilient type" for lazy var. Is this expected?